docs/source/dev_guide/review_process.rst

```diff
@@ -0,0 +1,95 @@
+to be polished at that stage. The difference between a Draft PR and a normal PR
+is minor. The Draft stage signals that your contribution is not ready for review
+but up for discussion. But it is nevertheless treated by like a normal PR in the
+sense that all the continuous integration tools are applied and there is an
+immediate feedback from automatic tests. Do not hesitate to contact us at any
+stage if you are unsure about anything.
+
+Once you have added something to the code base, please check locally if the
+tests pass. These tests include the code tests as well as the tests for code
+style as far as they can be checked automatically. You should also add tests for
+new functionality. Likewise, if your changes break existing tests, be sure to
+update (not deactivate!) the affected tests so that they pass. The next step is
+to merge or rebase with the master branch (if you are unsure about which option,
+do not hesitate to contact us). If necessary, add changes to the code to comply
+with any other added tests coming in from the updated master branch if they were
+not resolved in the merging or rebase process. If your PR started out as a
+Draft, this is now the time to convert it to a normal PR. If the PR never went
+into a Draft PR this is the point where you would actually create the PR. Upon
+creating the PR the continuous integration tools run a set of extensive tests
+with different compilers and compiling modes, such as debug (mode), to ensure
+compatibility with a variety of pre-defined defined sets of options.
+
+Once the PR is ready for review (all items on the checklist are completed), use
+the Github button to notify specific developers to review the changes.
+
+To help speed up the process we follow some rough guidelines so everyone knows
+what to expect from a reviewer's role:
+
+ * The first person to leave a full review becomes the responsible reviewer for
+   that PR.
+
+ * Once the reviewers' concerns have been addressed, the same person is
+   responsible for approving them as soon as possible. This not only ensures
+   that the changes are available to everyone as soon as they are ready, but
+   also minimizes the need for further merges with master and associated extra
+   work.
+
+ * Single comments without actually providing a review are allowed and
+   encouraged, but should be used sparsely. They can be really helpful if
+   something might be overseen. But if you want changes and involve yourself in
+   the process, make a proper review and take the responsibility to see it
+   through.
+
+ * If you are a reviewer and you start adding changes yourself (excluding the
+   one-line propositions that Github provides or just cosmetics) another person
+   should become the reviewer. This is to ensure that any part is reviewed by
+   someone else other than the creator.
+
+A last note related to our own experience. If it happens that your PR falls into
```

Comment:
```suggestion
A last note related to our own experiences. If it happens that your PR falls into
```

Reply:
Again, "experience" was correct.

Reply:
Well this one did not change. I reverted the functionalities.

Reply:
Ok cool, I didn't see that the suggestion hadn't actually been committed.